d2renderers/d2svg/d2svg.go

```diff
@@ -39,10 +39,15 @@ const (
 	DEFAULT_PADDING            = 100
 	MIN_ARROWHEAD_STROKE_WIDTH = 2
 	threeDeeOffset             = 15
+
+	tooltipIconLen = 32
```

Comment:
`tooltipIconSize`?

Reply:
it's circular, and uses radius elsewhere, so length clarifies that

Reply:
The radius usage is not related to this one.
maybe, diameter then

Reply:
it is, it's replacing this icon